### ISSUE TITLE
Удаляет существующее значение 

### ISSUE THREAD

**Issue by Kossin7** [webinmd/mixedimage#39]
Приветствую!
Косяк заметил, хотя может это настройка какая то нужна
- Файл уже загружен
![image](https://github.com/webinmd/mixedimage/assets/10547130/4c031188-5c71-4e8a-9932-19f7dd791d48)

- Нажимаю "С вашего компьютера"

- Не выбираю файла, просто закрываю диалоговое окно выбора файла.

-  меняет значение ТВ, загружает автоматом пустоту
![image](https://github.com/webinmd/mixedimage/assets/10547130/2a7ff033-7fd9-49ec-84ff-d30c45a77bfb)

**Comment by webinmd** [comment]
да, такое происходит если после загрузки изображения сразу выбирать другое, без перезагрузки страницы, попробую исправить
